app/views/layouts/application.html.erb

```diff
@@ -20,7 +20,9 @@
           </div>
           <div class="nav-details">
             <div class="blogname">Kolosek</div>
-            <div class="username">Tatjana B.</div>
+            <div class="username">
+              <%= current_user.profile.fullname %>
```

Comment:
You can delegate profile attributes to user, so you can access full name with current_user.fullname
https://apidock.com/rails/Module/delegate

Reply:
also, please use full_name instead of fullname